docker/Dockerfile

```diff
@@ -1,20 +1,19 @@
 # This dockerfile is based on Bitrise but with a lot of extra cruft we don't need removed
 FROM ubuntu:focal
-MAINTAINER Parity Technologies <admin@parity.io>
+LABEL maintainer="Parity Technologies <admin@parity.io>"
```

Comment:
A good practice would be write more metadata as we do pretty much everywhere [else](https://github.com/paritytech/scripts/blob/master/dockerfiles/ci-linux/Dockerfile#L8)

Reply:
👍 I'll adjust it accordingly

Reply:
apparently, as long as you're not publishing it, and I've never found if you do, this is not necessary.

Reply:
yep, basically, this image is intended only for internal use, to be able to have a quick set up to build an Android version